Tools/ardupilotwaf/boards.py

```diff
@@ -368,6 +368,11 @@ def configure_env(self, cfg, env):
             env.CXXFLAGS += [
                 '-O3',
             ]
+        elif 'clang++' in cfg.env.COMPILER_CXX:
```

Comment:
Let's say "no" to double-negatives :-)  Please remove the `not` in the `if not`

Reply:
Fixed